haigha/transports/socket_transport.py

```diff
@@ -29,26 +34,33 @@ def connect(self, (host, port), klass=socket.socket):
         Connect assuming a host and port tuple.
         '''
```

Comment:
I'm 100% for documenting code like this. I just need @awestendorf to sign off on a doc format. The sphinx format you've suggested is fairly ubiquitous, but there are also other standards, like napolean, that achieve the same but look different. I see sphinx format used in the message potion of the code base, but it's so minimal I want to make sure that's actually a standard chosen by the project owner.